models/r1_rover/r1_rover.sdf.jinja

```diff
@@ -515,6 +515,15 @@
         <use_parent_model_frame>1</use_parent_model_frame>
       </axis>
     </joint>
+    <include>
+      <uri>model://aruco_marker</uri>
+      <pose>0 0 0 0 0 0</pose>
+      <name>aruco</name>
+    </include>
```

Comment:
I don't think we want the aruco marker added to the r1 rover by default, but probably we can add an additional model that supports it